### ISSUE TITLE
Creating batches of tokenized text

### ISSUE THREAD

**Issue by achen353** [lucidrains/DALLE-pytorch#33]
How do you go about stacking unequal lengths of tokenized text to create batches of text for training?

**Comment by lucidrains** [comment]
@achen353 Hi Andrew, thanks for your interest! I'll be instrumenting the full training script this weekend :) stay tuned..

**Comment by achen353** [comment]
@lucidrains Thanks Phil that would be great. My GPU doesn't have PyTorch support yet so I'm working to implementing the model on TF. One idea I had was to create a fixed-length batch, fill the unused tail length of each sequence with 0 and make sure that their corresponding masks are "False".

**Comment by lucidrains** [comment]
Yup, that is the right way to go about it! I'll create all the necessary tools for training end to end without much coding, soon
